src/aiortc/rtcrtpsender.py

```diff
@@ -321,7 +330,7 @@ async def _run_rtp(self, codec: RTCRtpCodecParameters) -> None:
         timestamp_origin = random32()
         try:
             while True:
-                if not self.__track:
+                if not self.__track or not self.__enabled:
```

Comment:
OK, now here things start getting worrying. This means that if if `enabled` is false, we're not consuming media from the track, which completely changes the media flow. For example if your media source is a webcam, and you disable sending you're going to blow up your RAM because frames are going to accumulate, or am I wrong?

Reply:
Currently you'll find a very similar situation if you negotiate the reception of RTP and don't do anything with it. Memory will grow indefinitely as you don't consume the incoming media.

IMO if the user disables sending RTP then she must also stop the track, in order to avoid accumulating frames.

Reply:
I'm a bit confused about this topic. In browser I can do getUserMedia(), obtain tracks for mic and webcam and not use them anywhere, and they are not gonna produce any leak. This is real-time world. If something is producing audio units of video frames and nobody is consuming them then they should just be discarded.

Reply:
The case described by @jlaine corresponds to:
* You create a transceiver with a sender track.
* You don't send any traffic because sender is disabled.

In this case the frames in the track are not being consumed hence the track is accumulating them.

> If something is producing audio units of video frames and nobody is consuming them then they should just be discarded.

Yes, aiortc could, in case of being disabled, retrieve the frames from the track as if it was going to be sent, but simply discard it.

Reply:
This is not a concern anymore. Handled here https://github.com/aiortc/aiortc/pull/1055/commits/8faf5318ccb4a688813ae17503a6203761f893d7